docs/source/glossary.rst

```diff
@@ -0,0 +1,85 @@
+Glossary
+========
+
+.. include:: _include.rst
+
+.. glossary::
+
+   ABI
+      A JSON description of the external interface for a ABI
```

Comment:
Minor nitpick but think wording with something like this may be a bit more accessible to those new to scene: